src/assets/translations/en/main.json

```diff
@@ -69,6 +69,7 @@
     "degradedTransactionHistory": "Transaction history degraded",
     "degradedInfo": "Unable to fetch data for some accounts.",
     "success": "Success",
+    "happySuccess": "Success!",
```

Comment:
It's not just any success copy, it's double the happiness

Reply:
I love this